src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -111,7 +111,8 @@ function BaseSelectionList<TItem extends ListItem>(
                 });
 
                 // If disabled, add to the disabled indexes array
-                if (!!section.isDisabled || item.isDisabled) {
+                // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+                if (!!section.isDisabled || item.isDisabled || (item.isSelectable !== undefined && !item.isSelectable)) {
```

Comment:
Should we update to `item.isSelectable === false`

Reply:
Updated!